app/views/volunteers/_navbar.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+    <div class="navbar-nav">
+            <%= link_to "RTT Home", "https://www.rttriangle.org/", class: "nav-item nav-link" %>
```

Comment:
The link here should point to the `root_url` instead of hardcoding in a URL.

Also, none of the `link_to` links here seem to be using the `nav-item` class, so that can be deleted.